### ISSUE TITLE
YOW problems

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1164]
Problems with the YOW Chaosnet client:

- It's in SYSNET; TS YOW.  Is there a better place?
- It wants to connect to MIT-CCC, host number 16310.

See #916.

**Comment by larsbrinkhoff** [comment]
The server, a PDP-11 running Unix.  @hga, is this the CCC 11/45, previously with the Logo group?

```
;; No longer exists --sra 19 Feb 89
;;HOST : CHAOS 16310 : CCC,MITCCC : PDP11 : UNIX : CHAOS/MAIL,CHAOS/TELNET,CHAOS/NAME,CHAOS/FINGER,CHAOS/TIME,CHAOS/FILE,CHAOS/SUPDUP :

**Comment by hga** [comment]
Yes, it had the serial number 4xx, I vaguely remember it might have been
413 or 415.  Was the LOGO Lab's system running Small ITS, SITS,
surplused to what became the Concourse Computer Center, CCC,
decommissioned after CCC got ... a surplus 11/70 and/or a surplus VAX
750 as I vaguely recall.  Ran first UNIX V7, then BSD 2.x.
- Harold

----- Original message -----
From: Lars Brinkhoff <notifications@github.com>
Date: Sat, 13 Oct 2018 21:56:37 -0700

The server, a PDP-11 running Unix. @hga[1], is this the CCC 11/45,
previously with the Logo group?


;; No longer exists --sra 19 Feb 89 ;;HOST : CHAOS 16310 : CCC,MITCCC :
;; PDP11 : UNIX : CHAOS/MAIL,CHAOS/TELNET,CHAOS/NAME,CHAOS/FINGER,CHAOS-
;; /TIME,CHAOS/FILE,CHAOS/SUPDUP :— You are receiving this because you were mentioned. Reply to this email
directly, view it on GitHub[2], or mute the thread[3].

Links:

  1. https://github.com/hga
  2. https://github.com/PDP-10/its/issues/1164#issuecomment-429596737

**Comment by larsbrinkhoff** [comment]
Thank you hga.  Maybe in that case this PDP-11 was the 11/70 since the host table says it's running Unix.

In any event, it's interesting that the AI Lab relied on CCC to provide the mission critical YOW server. ;-)

**Comment by hga** [comment]
> Thank you hga. Maybe in that case this PDP-11 was the 11/70 since the host table says it's running Unix.

As I understand it, an 11/70 is just an 11/45 with bigger, faster bus, a 2KiB cache, and a Massbus.  The initial 11/45 that the CCC started with only ran UNIX; while I looked a bit at SITS, we didn't try to run it.

**Comment by leighklotz** [comment]
I don't recall the SITS 11 being on Chaosnet by the time I arrived, which was when CADR-6 was the newest.   The 11 was pretty much off by then, and I don't know who would have done the work or why.

**Comment by larsbrinkhoff** [comment]
@lispwright, you remembered the Logo 11/45 serial number as 413 or 415.  How do you feel about 422?  I don't know if the below refers specifically to the (formerly) Logo 11/45, but it seems possible.

```
\Dear{Ms. Biren,}

The \ccc\ would like to request a UNIX Version Seven license
under the MIT general license.  The \ccc\ is a cooperative
program combining the efforts of the Concourse Program and
the Logo Project.  Concourse is an alternative first year
study program for MIT undergraduates, with a small group of
faculty teaching the standard first-year subjects to a
group of approximately fifty freshmen.  The Logo Project has
been involved for ten years in the study of the uses of
technology to enhance education.  Emphasis has been placed
on providing simple and powerful languages and systems to
enhance contact with mathematics and science and also to
promote the general intellectual development of
pre-schoolers to college undergraduates.

We propose to provide high quality timesharing service to
the members of these programs, concentrating in the areas of
education and education research.  The Logo Project will use
the machine to continue their educational research.  The
faculty of the Concourse Program will use it to help teach
physics, calculus, and other subjects, and the students will
do homework and individual projects.

The machine is a PDP-11/45, serial number 422, located in
20B-031, MIT.  The research is purely educational, that of
Concourse's funded by its budget from the Institute, and
that of the Logo's mostly funded by the National Science
Foundation.  The results of this research will be in the
public domain.

Thank you for your consideration.
```

**Comment by lispwright** [comment]
> @lispwright, you remembered the Logo 11/45 serial number as 413 or 415. How do you feel about 422? I don't know if the below refers specifically to the (formerly) Logo 11/45, but it seems possible.

This letter, not sure who drafted it but it could well have been myself since I was handling a lot of this sort of administrative stuff, would have referred to the Logo Lab 11/45 given the context including apparent timing of the letter, and as far as I know we never procured  another 11/45.

No idea about the correct serial number, since I don't recall actually looking at the CPU unit for the number.  All of these numbers do point to it being one if the earliest manufactured, if the general understanding at the time that DEC started at 400 was correct.

**Comment by larsbrinkhoff** [comment]
@lispwright Thanks!
